pkg/api/check_target.go

```diff
@@ -40,6 +40,6 @@ func checkTargetHandler(request *http.Request) (*executor.Command, error) {
 		return nil, err
 	}
 
-	c := executor.New("remote-target-check-group", actorInput)
+	c := executor.New("remote-target-check-group-v2", actorInput)
```

Comment:
Why do you want to add ```-v2```? Can't you change actual actor? Is it because something different is using remote-target-check-group "v1"?